hotsos/core/plugins/openvswitch.py

```diff
@@ -204,3 +206,65 @@ def get_results_stats(self, results, key_by_date=True):
     def summary(self):
         # mainline all results into summary root
         return self.run_checks()
+
+
+class OVNChecksBase(OpenvSwitchChecksBase, plugintools.PluginPartBase):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.ovn_package_names = ['ovn-central', 'ovn-host']
+        self.certificate_expire_days = 60
+
+    @property
+    def ovn_type(self):
```

Comment:
Since this is used to determine if ovn is installed I think it would be better to replace that with a check like is ovn-common package installed and/or is ovn-host/ovn-northd etc systemd service enabled. That could be done entirely in the yaml check and therefore not require this python code. Also, ovn uses a number of certs for communication between different ovn services and we should be sure that we are checking them all.